lib/reference/mediaFeatures.js

```diff
@@ -6,25 +6,27 @@ const deprecatedMediaFeatureNames = new Set([
 	'device-aspect-ratio',
 	'device-height',
 	'device-width',
-	'max-device-aspect-ratio',
-	'max-device-height',
-	'max-device-width',
-	'min-device-aspect-ratio',
-	'min-device-height',
-	'min-device-width',
 ]);
 
-const rangeTypeMediaFeatureNames = new Set([
+const rangeTypeMediaFeatureNames = uniteSets(deprecatedMediaFeatureNames, [
 	'aspect-ratio',
-	'color-index',
 	'color',
+	'color-index',
 	'height',
+	'horizontal-viewport-segments',
 	'monochrome',
 	'resolution',
+	'vertical-viewport-segments',
 	'width',
 ]);
 
-const mediaFeatureNames = uniteSets(deprecatedMediaFeatureNames, rangeTypeMediaFeatureNames, [
+const rangeTypeMediaFeatureNamesWithMinMaxPrefix = new Set(
+	[...rangeTypeMediaFeatureNames].flatMap((name) => {
+		return [`min-${name}`, `max-${name}`];
+	}),
+);
```

Comment:
I like have this automatically follow `rangeTypeMediaFeatureNames` but I don't mind changing this to a hard coded list.

Reply:
Does `min-horizontal-viewport-segments` and `max-horizontal-viewport-segments` exist?
If not can you add a test for `max-horizontal-viewport-segments`?

Reply:
`min-` and `max-` prefixes are required to work in browsers for all `range` type features :

- https://www.w3.org/TR/mediaqueries-5/#mq-min-max
- https://www.w3.org/TR/mediaqueries-5/#media-descriptor-table

> `horizontal-viewport-segments` `<integer>` `range`

-----

Is there anything specific about `horizontal-viewport-segments` that would make it interesting to have extra test coverage?

From a syntax/grammar standpoint this is the same as the `color` feature.

Reply:
Here's [some context](https://github.com/w3c/csswg-drafts/issues/5621#issuecomment-713142655) for the ones interested.